cmd/experimental/kjobctl/pkg/cmd/list/list_test.go

```diff
@@ -90,15 +91,16 @@ ns2         j2     profile2   lq1           3/3           60m        60m
 					Profile("profile2").
 					LocalQueue("lq2").
 					JobStatus(rayv1.JobStatusSucceeded).
+					RayClusterName("cq1").
```

Comment:
maybe we should have one with no cluster set, (I think that while the job is suspended it's empty)

Reply:
You right, it's a good point. Done.